pyproject.toml

```diff
@@ -3,9 +3,7 @@ name = "ls-pre-commit-hooks"
 version = "0.7.32"
```

Comment:
Bump the version here as well.

Reply:
I guess it doesn't really matter as we release with github tags, but still, it's out of sync :P